packages/sources/src/components/selectable/selectable.models.ts

```diff
@@ -4,7 +4,7 @@ export interface Selectable<TemplateFnReturnType> {
   type: "radio" | "checkbox";
   id: string;
   name?: string;
-  label: string;
+  label: string | TemplateFnReturnType;
```

Comment:
Dit lijkt mij een ongewenste wijziging. Deze property wordt direct in `<label>` geplakt, ik denk niet dat we daar enige ruimte willen geven.

Reply:
Dan kunnen we de `dso-badge` die hier in het label geplakt wordt niet doen: https://dso-toolkit.nl/45.2.0/components/detail/zoeken-in-lijst-cards.html

Reply:
OK, duidelijk